src/user/authorization/jwtUtils.ts

```diff
@@ -1,12 +1,12 @@
 import jwt from "jsonwebtoken";
-import { z } from "zod";
-import { JwtSchema } from "../userSchema";
-
-type UserInfo = z.infer<typeof JwtSchema>;
+import { UserInfo } from "../userSchema";
+import redisClient from "../../redis";
 
 const SECRET_KEY = process.env.JWT_SECRECT_KEY || ("secret" as string);
 const AT_EXPIRED = process.env.AT_EXPIRE || ("1m" as string);
-const RT_EXPIRED = process.env.RT_EXPIRE || ("3d" as string);
+const RT_EXPIRED = process.env.RT_EXPIRE || ("3m" as string);
+const RT_EXPIRED_BY_NUMBER =
+  60 * 60 * 24 * Number(process.env.RT_EXPIRED_BY_NUMBER);
```

Comment:
```suggestion
  60 * 60 * 24 * Number(process.env.RT_EXPIRED_BY_NUMBER || 1);
```
`process.env`에 `RT_EXPIRED_BY_NUMBER`가 정의되지 않았을 시 #76 과 같은 오류가 발생합니다. 추후에 환경 변수를 zod 스키마로 파싱하고 기본값을 주는 식으로 바꾸면 될 것 같아요!

Reply:
아하 이 부분을 제가 놓쳤었군요 ㅠ 확인 감사합니다!